### PR TITLE
[xray] Fixed broken joinkey condition logic

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by rahulsadanandan — APPROVED** [review]

File: stable/xray/UPGRADE_NOTES.md [hunk 1]
```diff
@@ -57,11 +65,11 @@ This file describes special upgrade notes needed at specific versions
...
+          $ helm install xray-new --set router.livenessProbe.enabled=false --set router.readinessProbe.enabled=false --set indexer.livenessProbe.enabled=false --set analysis.livenessProbe.enabled=false --set server.livenessProbe.enabled=false --set persist.livenessProbe.enabled=false --set indexer.readinessProbe.enabled=false --set analysis.readinessProbe.enabled=false --set server.readinessProbe.enabled=false --set persist.readinessProbe.enabled=false --set postgresql.enabled=false --set database.user=<OLD_PG_USERNAME> --set database.password=<OLD_PG_PASSWORD> --set database.url="postgres://<SERVICE_NAME_POSTGRES>:5432/xraydb?sslmode=disable" --set xray.mongoUsername=<OLD_MONGO_USERNAME> --set xray.mongoPassword=<OLD_MONGO_PASSWORD> --set xray.mongoUrl="mongodb://<SERVICE_NAME_MONGODB>:27017/?authSource=xray&authMechanism=SCRAM-SHA-1" --set xray.masterKey=<PREVIOUS_MASTER_KEY>  --set rabbitmq.auth.password=<PASSWORD> --set rabbitmq.enabled=true --set rabbitmq-ha.enabled=false --set xray.jfrogUrl=<NEW_ARTIFACTORY_URL> --set  xray.joinKey=<JOIN_KEY>
```
Comment (eldada) [inline stable/xray/UPGRADE_NOTES.md:68]: We should move these blocks to be yaml file examples. It's hard to understand it.

**Review by eldada — APPROVED** [review]

File: stable/xray/UPGRADE_NOTES.md (hunk 1 above)
Comment (rahulsadanandan) [reply stable/xray/UPGRADE_NOTES.md]: Changed.

**Comment by rahulsadanandan** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by chukka** [comment]
recheckcla

**Pull request merged**